test/integ/main_test.go

```diff
@@ -13,353 +13,246 @@
 // limitations under the License.
 
 // Main entry point for integration tests.
-package integ
+package integration
```

Comment:
This is really difficult to read, to double confirm, this is just renaming the main_new_test.go?

Reply:
Yes, we removed the old ci, and replaced it with the new ci, the old ci is in package `integ`, the new ci is in package `integration`